### DISCUSSION TITLE
Is there a way to add a biography to the author?

### DISCUSSION THREAD

**Discussion by Benjamin-Lee** [jpanther/congo#25] [Ideas]
On my personal site, I put a short blurb about myself at the bottom of the page for people who might stumble across a post from Google. I'm trying porting to Congo (after getting some constructive feedback on my site from HN) and noticed this doesn't seem supported. Is there a way I'm missing?

**Comment by jpanther** [comment]
Thanks for the comment. This isn't something that is currently supported but I can definitely add it to the next release.

**Reply by Benjamin-Lee** [reply]
Wow, that was fast. Thanks!

**Reply by jpanther** [reply]
You had very lucky timing as I was just about to publish. The change is now live in v1.5.0. Just add your text to the `bio` parameter in the author config.
